### DISCUSSION TITLE
Autoplay

### DISCUSSION THREAD

**Discussion by robotlovecoffee** [dimsemenov/PhotoSwipe#1975] [Q&A] [answered]
I do not see an option to have the image autoplay.  I'm looking to build a very basic gallery that just loops thru images.

**Comment by acwolff** [comment, marked as answer]
That is possible as you can see in [this album](https://andrewolff.jalbum.net/Vecht_Stegeren_PS_Carousel/), but you have to add the code yourself. I agree with you that it would be nice if Autoplay with transitions was implemented in the Photoswipe library, see my request #1875

**Reply by robotlovecoffee** [reply]
Thanks, I will look to do this as this will be running in a kiosk mode and I do not need interaction, but also want to use the same lib for mobile and desktop versions that would have interaction.
